### DISCUSSION TITLE
What is the capacity of Skywalking?

### DISCUSSION THREAD

**Discussion by SooryaMuthuraj** [apache/skywalking#11801] [General]
What is the Maximum count of Agents skywalking supports? also is root permission is required for installing skywalking components (UI) and Skywalking agent

**Comment by wu-sheng** [comment]
No, there isn't limitation for that. If you have large scale deployment, choose to run OAP in cluster mode, and setup Database powerful enough for that.

We don't require root, as UI and OAP are JVM based.

**Reply by SooryaMuthuraj** [reply]
There is no JVMs like Tomcat or Glassfish for Apache skywalking

**Reply by wu-sheng** [reply]
What does this mean?
